src/components/uploadSiteImageButton/index.tsx

```diff
@@ -102,6 +118,13 @@ const UploadSiteImageButton: React.FC<UploadImageProps> = ({ siteEntryId }) => {
         <ConfirmUpload onClick={onClickUploadSiteImage}>
           {t('uploadSiteImage.upload_button_message')}
         </ConfirmUpload>
+        <Checkbox
+          onChange={async (e: CheckboxChangeEvent) => {
```

Comment:
:question: Similar to the `beforeChange` async function above - this seemed to work without being async, did it make a difference for you?